clients/privacy-center/pages/consent.tsx

```diff
@@ -214,16 +221,61 @@ const Consent: NextPage = () => {
     redirectToIndex,
   ]);
 
+  /**
+   * Initialize internalization library.
+   */
+  const experience = useAppSelector(selectPrivacyExperience);
+  const { IS_OVERLAY_ENABLED } = settings;
+  const isConfigDrivenConsent = !IS_OVERLAY_ENABLED;
+  const [isI18nInitialized, setIsI18nInitialized] = useState(false);
+
+  useEffect(() => {
+    const i18n = setupI18n();
+
+    // 1. Config driven consent
+    if (isConfigDrivenConsent) {
+      // If we're in config driven consent then
+      // still load the static messages in order to have default english
+      // messages available
+      loadMessagesFromFiles(i18n);
+      setI18nInstance(i18n);
+      setIsI18nInitialized(true);
+      return;
+    }
+
+    // 2. Notice driven consent
+    if (!experience) {
+      return;
+    }
+
+    initializeI18n(
+      i18n,
+      window?.navigator,
+      experience as PrivacyExperience,
+      {
+        debug: true,
+      },
+      {}
+    );
+
+    setI18nInstance(i18n);
+    setIsI18nInitialized(true);
+  }, [experience, setI18nInstance, isConfigDrivenConsent]);
+
   return (
     <Stack as="main" align="center" data-testid="consent">
-      <Stack align="center" py={["6", "16"]} spacing={8} maxWidth="720px">
-        <Stack align="center" spacing={3}>
-          <ConsentHeading />
-          <ConsentDescription />
+      {/* Wait until i18n is initalized so we can diplay the correct language and
+       also we can use the correct history ids */}
+      {isI18nInitialized ? (
```

Comment:
```suggestion
      {isI18nInitialized && (
```

Reply:
Yup, I like just using && to. Jeremy mentioned something about false-y values that can create confusion, and that's why they prefer the ? and null: But this is a boolean, so no confusion there. I'll apply the suggestions.